pkg/apiserver/certificate/certificate.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package certificate
+
+import (
+	"fmt"
+	"net"
+
+	"k8s.io/apiserver/pkg/server/dynamiccertificates"
+	"k8s.io/apiserver/pkg/server/options"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/util/cert"
+	"k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset"
+)
+
+var (
+	// certDir is the directory that the TLS Secret should be mounted to. Declaring it as variable for testing.
+	certDir = "/var/run/antrea/antrea-controller-tls"
+	// The DNS names that the TLS certificate will be signed with.
+	// Note that antrea-agent and kube-aggregator only verify the short server name.
+	// The second server name may not work if user specify a different cluster domain.
```

Comment:
why is the second server name needed then? if it's needed should we have a TODO / issue to support user-provided cluster domain names?

Reply:
Tried to sign it with the default cluster domain, removed it as we don't use it for now. 
We could see whether there's really such requirement that the service is accessed via its full FQDN and add it when needed.